apps/camera/js/views/preview-gallery.js

```diff
@@ -27,7 +27,7 @@ require('gaia-header');
 var SWIPE_DISTANCE_THRESHOLD = window.innerWidth / 3; // pixels
 var SWIPE_VELOCITY_THRESHOLD = 1.0;                   // pixels/ms
 
-var SWIPE_DURATION = 250;   // How long to animate the swipe
+var SWIPE_DURATION = 150;   // How long to animate the swipe
```

Comment:
This was purposely set long enough to hide some kind of image decoding ugliness. But if a smaller value works well, please do change. it.